softwareconfiguration/model/src/main/java/ai/libs/softwareconfiguration/model/Util.java

```diff
@@ -0,0 +1,286 @@
+package ai.libs.softwareconfiguration.model;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Deque;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.commons.math3.geometry.euclidean.oned.Interval;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ai.libs.jaicore.basic.sets.Pair;
+
+public class Util {

```

Comment:
Util for what?

Reply:
I added a ticket for this into the Sprint to consider this change after the PR.